src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.props

```diff
@@ -31,6 +31,11 @@ Copyright (c) .NET Foundation. All rights reserved.
     <None Remove="**/*$(DefaultLanguageSourceExtension)" Condition=" '$(EnableDefaultCompileItems)' == 'true' "/>
     <None Remove="**/*.resx" Condition=" '$(EnableDefaultEmbeddedResourceItems)' == 'true' "/>
   </ItemGroup>
+  <ItemGroup Condition=" '$(EnableDefaultDependentUponMetadata)' == 'true' And '$(EnableDefaultResxDependentUponMetadata)' == 'true' ">
+    <Compile Update="**/*.Designer.cs" DependentUpon="$([System.IO.Path]::ChangeExtension($([System.IO.Path]::GetFileNameWithoutExtension('%(Identity)')), '.resx'))" />
+    <EmbeddedResource Update="**/*.??.resx" DependentUpon="$([System.IO.Path]::ChangeExtension($([System.IO.Path]::GetFileNameWithoutExtension('%(Identity)')), '.resx'))" />
+    <EmbeddedResource Update="**/*.??-??.resx" DependentUpon="$([System.IO.Path]::ChangeExtension($([System.IO.Path]::GetFileNameWithoutExtension('%(Identity)')), '.resx'))" />
```

Comment:
This is not correct. There are cultures like zh-Hant that match neither of these.

Reply:
It is perfectly legal to have only culture-specific resx files and no culture-neutral. Another case where existence matters. I'm not sure we should ever leave the evaluation with dangling DependentUpon

Reply:
Another issue is, for example, MyProduct.MyStrings.resx where MyStrings is not a culture.

Reply:
> It is perfectly legal to have only culture-specific resx files and no culture-neutral. Another case where existence matters. I'm not sure we should ever leave the evaluation with dangling DependentUpon

It's been updated to include a working condition for only existent files.

> Another issue is, for example, MyProduct.MyStrings.resx where MyStrings is not a culture.

This seems like the kind of nesting users may want by default, but we can certainly drop the `*.*.resx` matching as default or even in general.

Reply:
I think the existence check covers this. If there's both MyProduct.resx and MyProduct.MyStrings.resx, then the nesting should be fine.